cluster-up/cluster/kind-1.19-sriov/certcreator/certsecret.go

```diff
@@ -120,7 +120,7 @@ func createSecret(clusterApi kubernetes.Interface, namespace, secretName string,
 }
 
 func main() {
-	namespace := flag.String("namespace", "", "The namespace of the webhook")
+	namespace := flag.String(strings.Join("namespace", "", "The namespace of the webhook"))
```

Comment:
This doesn't look right, probably needs to be reverted.

Reply:
Done